lib/cdo/script_constants.rb

```diff
@@ -141,7 +143,11 @@ def self.position_in_category(script, category)
   end
 
   def self.category_priority(category)
-    CATEGORY_ORDERING_PRIORITY[category.to_sym] || 0
+    if category == OTHER_CATEGORY_NAME
+      1e6 # 'other' goes at the end
```

Comment:
Could be something like `CATEGORIES.keys.length` which perhaps makes it a little clearer (and also protects us against the case where we literally have a million categories :-P)